joplin/base/models/janis_page.py

```diff
@@ -129,14 +129,23 @@ def janis_url(self):
                     except AttributeError as e:
                         # this is for pages just under departments
                         theme_slug = self.related_departments.all()[0].related_department.slug or None
-
+                    finally:
```

Comment:
def some tech debt here; repeating the same code twice is ugly, but it needed to be under this separate conditional in order for all use cases to be covered. I tried writing a function to call that would encapsulate this, but it didn't really make much sense. Open to ideas/future refactoring

Reply:
Why is copy + pasting into "finally" required? Wouldn't the code just keep going and create the paths list?

The double nested Attribution Error catching seems weird. Is there a clearer way to express what's happening? Like, should each Attribution Error only happen for a particular content type?

Reply:
^ 2nd part isn't part of this issue, feel free to ignore.

Reply:
@Niicck actually I think the 2nd is related to first. I must have assumed it would keep going when I first wrote it, but it dosen't. Probably because they are nested inside that 'if' statement